java/pom.xml

```diff
@@ -46,7 +46,7 @@
     </issueManagement>
 
     <properties>
-        <java.version>17</java.version>
+        <java.version>11</java.version>
```

Comment:
Does this have an affect on what the compiled code looks like? What is the impact of this?

Reply:
I didn't notice any differences between the two builds.

[jdk11.log](https://github.com/extism/extism/files/10381062/jdk11.log)
[jdk17.log](https://github.com/extism/extism/files/10381063/jdk17.log)